docusaurus/docs/Android/04-compose/04-message-components/03-message-list.mdx

```diff
@@ -111,82 +135,82 @@ We allow for a few ways of customizing the `MessageList` component, as per the s
 ```kotlin
 @Composable
 fun MessageList(
-	..., // State
+	..., // State and actions
     modifier: Modifier = Modifier,
-    loadingContent: @Composable () -> Unit = { LoadingView(modifier) },
-    emptyContent: @Composable () -> Unit = { DefaultMessageListEmptyView(modifier) },
-    itemContent: @Composable (MessageListItem) -> Unit = {
-        DefaultMessageItem(
-            messageListItem = it,
-            onLongItemClick = onLongItemClick,
+    loadingContent: @Composable () -> Unit = { DefaultMessageListLoadingIndicator(modifier) },
+    emptyContent: @Composable () -> Unit = { DefaultMessageListEmptyContent(modifier) },
+    itemContent: @Composable (MessageListItemState) -> Unit = { messageListItem ->
+        DefaultMessageContainer(
+            messageListItem = messageListItem,
+            onImagePreviewResult = onImagePreviewResult,
             onThreadClick = onThreadClick,
-            onImagePreviewResult = onImagePreviewResult
+            onLongItemClick = onLongItemClick,
+            onReactionsClick = onReactionsClick,
+            onGiphyActionClick = onGiphyActionClick
         )
     },
 )
 ```
 
 * `modifier`: Modifier for the root component. Useful for things like the component size, padding, background and similar.
-* `loadingContent`: Customizable composable function that allows you to override the default `LoadingView` when loading the initial data set.
-* `emptyContent`: Customizable composable function that allows you to override the empty state, when there are no messages available.
-* `itemContent`: Composable function that allows you to fully override the UI and behavior of each message in the list. This function will be applied to each item in the list and you'll gain access to the `MessageListItem` inside the lambda when building your custom UI.
+* `loadingContent`: Composable function used while loading the message data. By default shows a loading indicator that you can override.
+* `emptyContent`: Composable function called when loading is finished and no messages are available. By default displays a message that you can override.
```

Comment:
I would say `an empty placeholder with a message` or just `an empty placeholder`

Reply:
We should also shorten all mentions of "Composable function" to jsuzt Composable. More less all composables are functions - no need for the redundancy

Reply:
Removed Composable too where unnecessarry. 

We have a code snippet above in which the functions are visibly annotated with @Composable so repeatedly mentioning these are Composables may be redundant as well.